lib/event_store_client/event.rb

```diff
@@ -8,9 +8,10 @@ module EventStoreClient
   class Event < Dry::Struct
     attr_reader :id
 
+    attribute :type, Types::Strict::String
+    attribute :title, Types::Strict::String.optional.default(nil)
```

Comment:
Does it work? `Types::Strict::String` should return error for nil. But maybe for optional it's not a case. Just please confirm if I'm wrong.

Reply:
```
EventStoreClient::Event.new(type: 'MyType')     
 => #<EventStoreClient::Event type="MyType" title=nil data="{}" metadata="{\"created_at\":\"2019-12-27 15:37:34 +0100\"}"> 
```